qutip/stochastic.py

```diff
@@ -105,7 +105,7 @@ def stochastic_solvers():
         Scheme keeping the positivity of the density matrix. (smesolve only)
         -Order strong 1.0?
         -Code: 'rouchon', 'Rouchon'
-        Eq. 4 of arXiv:1410.5345 with eta=1 
+        Eq. 4 of arXiv:1410.5345 with eta=1
```

Comment:
Please add published reference too: Phys. Rev. A 91, 012118, (2015).

Reply:
This occurs in the mcsolve PR but seems related to the other solvers.

Reply:
Updated it in #990.
Change in 990 are included in 991, then 969 include both.
master -> 990 -> 991 -> 969